test_runner/src/main/kotlin/ftl/ios/xctest/common/InstallParseBinaries.kt

```diff
@@ -54,8 +55,16 @@ private fun downloadAndUnzip(osname: String) {
             it.setExecutable(true)
         }
     Files.delete(destinationFile)
-    createSymbolicLinkToFile(
-        link = Paths.get(flankBinariesDirectory.toString(), "libatomic.so.1"),
-        target = Paths.get(flankBinariesDirectory.toString(), "libatomic.so.1.2.0")
-    )
+
+    if (isWindows) {
```

Comment:
we cannot create a symobolic link on native windows without admin privalages.

Reply:
yes, but we could make a copy  `if (windows)` so there will be just single method
`createSymbolicLink` which has inside copy implementation for windows and symbolic link for others

Reply:
👍